packages/sdk/test/integration/history_tree_split_test.ts

```
@@ -1249,31 +1251,61 @@ describe('Tree History - multi client split L2 edge cases', () => {
+    return [
+      ...(op?.getRestoreSpans() ?? []),
+      ...(op?.getRetombstoneSpans() ?? []),
+    ];
+  }
+
+  // An ordered, identity-bearing description of a reverse payload. Comparing
+  // sizes alone would not notice a return to copy-reinsert, which can emit the
+  // same number of spans while minting a fresh identity every cycle -- exactly
+  // the regression identity-preserving restore exists to prevent.
+  function fingerprint(op: TreeEditOperation | undefined): Array<string> {
+    return reversePayload(op).map((s) =>
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)